tests/data/mock.4stats.pairs

```diff
@@ -0,0 +1,21 @@
+## pairs format v1.0.0
```

Comment:
Thanks for dedup stats! It's great. Should we also add cases with the same coordinates but with different orientations?